lib/isViolateRule.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+    isViolateRule: (journeyName) => {
```

Comment:
perhaps we could rename this to be clearer:  isValidJourneyName

Reply:
I will rename it on the ticket am working on now SAL, Thank you

Reply:
👍